schemas/user.yaml

```diff
@@ -13,6 +13,8 @@ user:
       type: string
     position:
       type: string
+    language:
```

Comment:
I am not sure if we should use the `user` object here. Isn't it used for other user returns as well? Then I think we need a new `current_user.yaml` file with a new `current_user` object which is returned by the user#get endpoint